src/wallet/rpcwallet.cpp

```diff
@@ -3576,7 +3576,7 @@ UniValue createrawpegin(const JSONRPCRequest& request)
         std::vector<unsigned char> witnessBytes(ParseHex(request.params[2].get_str()));
         witnessProgScript = CScript(witnessBytes.begin(), witnessBytes.end());
         if (!witnessProgScript.IsWitnessProgram(version, witnessProgram) || version != 0) {
-            throw JSONRPCError(RPC_INVALID_PARAMETER, "Given witness_program is not a valid v0 witness program.");
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "Given claim_script is not a valid v0 witness program.");
```

Comment:
hmm yes, originally this made sense since it was `claimpegin` functionality which currently assumes this, but it might not make sense for the raw rpc call. Will fix.